app/routes/_gcn.circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -264,19 +246,8 @@ export function CircularEditForm({
             name="intent"
             value="save"
           >
-            {circular ? 'Update' : 'Send'}
+            Send
           </Button>
-          {/* TODO: implement this
-          {circular && (
-            <Button
-              disabled={sending || !valid}
-              type="submit"
-              name="intent"
-              value="distribute"
-            >
-              Update and Distribute
-            </Button>
-          )} */}
```

Comment:
Remove commented out code.

Reply:
That should have been removed in the last commit